modules/sor/sor.prisma

```diff
@@ -0,0 +1,14 @@
+model PrismaTradeResult {
```

Comment:
See other comment, would remove this model and just log this data to cloudwatch, its possible to run queries on the data.